### PR TITLE
Updated tower_db_url example

### PR DESCRIPTION
**Pull request by justinegeffen**
Updated example to reflect the breaking change parameter in changelog.

### CODE REVIEW

**Comment by jason-seqera** [comment]
@justinegeffen, I think we'll need to do all of these:

```
6 results - 2 files

platform_versioned_docs/version-24.1/enterprise/configuration/configtables/data_features_env.yml:
   4    Description: > 
   5:     Enable [Data Explorer](https://docs.seqera.io/platform/24.1.0/data/data-explorer) in all workspaces. To mount data inside a data studio, you must enable Data Explorer.
   6    Value:                'Default: `false`'

  34    Description: >
  35:     The URL of the Data Studios connect proxy. The connect proxy is used internally by Seqera Platform. See [Data Studios deployment](https://docs.seqera.io/platform/24.1.0/enterprise/data-studios).
  36    Value:                'Example: `https://connect.example.com/`'

  39    Description: > 
  40:     An access token used to register new clients in Seqera Platform. Any alphanumeric value is allowed. See [Data Studios deployment](https://docs.seqera.io/platform/24.1.0/enterprise/data-studios).
  41    Value:                'd5XDoRzHpWo1c............mDnfBpB'

  44    Description: > 
  45:     The file path to a PEM certificate used to sign the OIDC tokens for the OpenID connect provider. See [Data Studios deployment](https://docs.seqera.io/platform/24.1.0/enterprise/data-studios).
  46    Value:                'Example: `/data-studios-rsa.pem`'

platform_versioned_docs/version-24.1/enterprise/configuration/configtables/features_env.yml:
  14    Description: > 
  15:     Enable Seqera configuration value retrieval from [AWS Parameter Store](https://docs.seqera.io/platform/24.1.0/enterprise/configuration/aws_parameter_store). 
  16    Value:                'Default: `false`'

  29    Description: > 
  30:     Enable the [stepped launch form](https://docs.seqera.io/platform/24.1.0/launch/launchpad#launch-form) in the workspaces specified. Set empty (`TOWER_STEPPED_LAUNCH_FORM_ALLOWED_WORKSPACES=`) to enable the new launch form in all workspaces, or provide a comma-separated list of workspace IDs to enable the form per workspace. 
  31    Value:                'Example: `976xxxxxxxx604,402xxxxxxxx677`'
```

Since each would need to be `24.1` now.

**Comment by llewellyn-sl** [comment]
Strange how these URLs didn't show up in my Find in Files search in VS Code. Seems all the JSON table hard-coded URLs were missed. Would be good to give all the config tables a quick scan to see if any other hard-coded URLs are present.

**Comment by justinegeffen** [comment]
> @justinegeffen, I think we'll need to do all of these:
> 
> ```
> 6 results - 2 files
> 
> platform_versioned_docs/version-24.1/enterprise/configuration/configtables/data_features_env.yml:
>    4    Description: > 
>    5:     Enable [Data Explorer](https://docs.seqera.io/platform/24.1.0/data/data-explorer) in all workspaces. To mount data inside a data studio, you must enable Data Explorer.
>    6    Value:                'Default: `false`'
> 
>   34    Description: >
>   35:     The URL of the Data Studios connect proxy. The connect proxy is used internally by Seqera Platform. See [Data Studios deployment](https://docs.seqera.io/platform/24.1.0/enterprise/data-studios).
>   36    Value:                'Example: `https://connect.example.com/`'
> 
>   39    Description: > 
>   40:     An access token used to register new clients in Seqera Platform. Any alphanumeric value is allowed. See [Data Studios deployment](https://docs.seqera.io/platform/24.1.0/enterprise/data-studios).
>   41    Value:                'd5XDoRzHpWo1c............mDnfBpB'
> 
>   44    Description: > 
>   45:     The file path to a PEM certificate used to sign the OIDC tokens for the OpenID connect provider. See [Data Studios deployment](https://docs.seqera.io/platform/24.1.0/enterprise/data-studios).
>   46    Value:                'Example: `/data-studios-rsa.pem`'
> 
> platform_versioned_docs/version-24.1/enterprise/configuration/configtables/features_env.yml:
>   14    Description: > 
>   15:     Enable Seqera configuration value retrieval from [AWS Parameter Store](https://docs.seqera.io/platform/24.1.0/enterprise/configuration/aws_parameter_store). 
>   16    Value:                'Default: `false`'
> 
>   29    Description: > 
>   30:     Enable the [stepped launch form](https://docs.seqera.io/platform/24.1.0/launch/launchpad#launch-form) in the workspaces specified. Set empty (`TOWER_STEPPED_LAUNCH_FORM_ALLOWED_WORKSPACES=`) to enable the new launch form in all workspaces, or provide a comma-separated list of workspace IDs to enable the form per workspace. 
>   31    Value:                'Example: `976xxxxxxxx604,402xxxxxxxx677`'
> ```
> 
> Since each would need to be `24.1` now.

These are addressed in https://github.com/seqeralabs/docs/pull/177/ but I've made it latest and not 24.1

File: platform_versioned_docs/version-24.1/enterprise/configuration/configtables/req_env_vars.yml [hunk 1]
```diff
@@ -14,8 +14,8 @@
 - 
   Environment variable:            '`TOWER_DB_URL`'
   Description: > 
-    The URL to access your database. **For installation in a new environment, this value must be set as an environment variable.**
-  Value:               'Example: `jdbc:mysql://db:3306/tower`'
+    The URL to access your database. **For installation in a new environment, this value must be set as an environment variable.** See the [24.1 release notes](https://docs.seqera.io/platform/24.1/enterprise/release_notes/enterprise_latest#breaking-changes) for information about the format of the URL.
```
Comment (llewellyn-sl) [inline platform_versioned_docs/version-24.1/enterprise/configuration/configtables/req_env_vars.yml:17]:
```suggestion
    The URL to access your database. **For installation in a new environment, this value must be set as an environment variable.** See the [24.1 release notes](https://docs.seqera.io/platform/24.1/enterprise/release_notes/enterprise_latest#breaking-changes) for information about the DB URL format.
```

**Review by christopher-hakkaart — APPROVED** [review]
LGTM

**Pull request merged**